src/networkcloud/azext_networkcloud/operations/custom_properties.py

```diff
@@ -0,0 +1,198 @@
+            = AAZObjectType()
+
+        operation_status_result_read = _schema_operation_status_result_read
+        operation_status_result_read.end_time = AAZStrType(
+            serialized_name="endTime",
+        )
+        operation_status_result_read.error = AAZObjectType()
+        cls._build_schema_error_detail_read(operation_status_result_read.error)
+
+        operation_status_result_read.properties = AAZObjectType()
+        cls._build_schema_properties_read(
+            operation_status_result_read.properties)
+
+        operation_status_result_read.id = AAZStrType()
+        operation_status_result_read.name = AAZStrType()
+        operation_status_result_read.operations = AAZListType()
+        operation_status_result_read.percent_complete = AAZFloatType(
+            serialized_name="percentComplete",
+        )
+        operation_status_result_read.resource_id = AAZStrType(
+            serialized_name="resourceId",
+            flags={"read_only": True},
+        )
+        operation_status_result_read.start_time = AAZStrType(
+            serialized_name="startTime",
+        )
+        operation_status_result_read.status = AAZStrType(
+            flags={"required": True},
+        )
+
+        operations = _schema_operation_status_result_read.operations
+        operations.Element = AAZObjectType()
+        cls._build_schema_operation_status_result_read(operations.Element)
+
+        _schema.end_time = cls._schema_operation_status_result_read.end_time
+        _schema.error = cls._schema_operation_status_result_read.error
+        _schema.properties = cls._schema_operation_status_result_read.properties
+        _schema.id = cls._schema_operation_status_result_read.id
+        _schema.name = cls._schema_operation_status_result_read.name
+        _schema.operations = cls._schema_operation_status_result_read.operations
+        _schema.percent_complete = cls._schema_operation_status_result_read.percent_complete
+        _schema.resource_id = cls._schema_operation_status_result_read.resource_id
+        _schema.start_time = cls._schema_operation_status_result_read.start_time
+        _schema.status = cls._schema_operation_status_result_read.status
+
+    # Custom handling of response will print out the output head and the result URL
+    # it will also save files into output directory if provided
+    def _output(self, *args, **kwargs):
+        args = self.ctx.args
+        properties = self.ctx.vars.instance.properties
+
+        # Print the output head to the console
+        if has_value(properties.outputHead):
+            print(properties.outputHead.to_serialized_data())
+            print("\n================================")
+
+        # Print out the result URL
+        if has_value(properties.resultUrl):
+            result_url = properties.resultUrl.to_serialized_data()
+            print(
+                "Script execution result can be found in storage account:\n" + result_url + "\n")
```

Comment:
If it's a warning message, please use `logger` instead of `print`. If should be in output stream, please return it.